requirements-vllm.txt

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+
+-r requirements.txt
+
+# newer vllm does not serve Merlinite
+vllm<0.4.3
```

Comment:
Instead of another requirements file, how about you use an optional dependency?

Add this to `pyproject.toml`
```toml
[project.optional-dependencies]
vllm = ["vllm<0.4.3"]
```
then you can install vllm support with `pip install instructlab[vllm]`